setup.cfg

```diff
@@ -67,22 +67,22 @@ dev =
     bandit==1.7.5
     black==23.3.0
     factory-boy==3.2.1
-    Faker==18.6.0
+    Faker==18.7.0
     flake8==6.0.0
-    hypothesis==6.75.1
+    hypothesis==6.75.2
     ipdb==0.13.13
     ipython==8.12.0
     isort==5.12.0
     logging-gelf==0.0.26
     memory-profiler==0.61.0
```

Comment:
I would remove:
- Faker
- ipdb
- ipython
- logging-gelf
- memory-profiler

as they are not used in the project, and don't seem to be used a lot by our devs (devs can still add a package and `make boostrap` again to add dev tools). WDYT?

Reply:
`logging_gelf` is necessary in the GELF logger fixture